### ISSUE TITLE
Error: Superclass has no method named 'forgetChild'.  

### ISSUE THREAD

**Issue by DhavalRKansara** [letsar/flutter_staggered_grid_view#129]
Getting below error while using this plugin.

<img width="1234" alt="Screen Shot 2020-07-30 at 1 59 49 PM" src="https://user-images.githubusercontent.com/44993081/88900383-f069fc00-d26c-11ea-9d30-eac7047cb8b8.png">

Flutter doctor:
```

[✓] Flutter (Channel unknown, v1.12.13+hotfix.9, on Mac OS X 10.15.5 19F101, locale en-US)
    • Flutter version 1.12.13+hotfix.9 at /Users/dhavalkansara/flutter
    • Framework revision f139b11009 (4 months ago), 2020-03-30 13:57:30 -0700
    • Engine revision af51afceb8
    • Dart version 2.7.2

 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at /Users/dhavalkansara/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 29.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.5)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.5, Build version 11E608c
    • CocoaPods version 1.9.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 3.6)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 45.0.1
    • Dart plugin version 192.7761
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)

[✓] IntelliJ IDEA Community Edition (version 2019.2.2)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    • Flutter plugin version 43.0.2
    • Dart plugin version 192.8052

[✓] Connected device (3 available)
    • vivo 1723  • 5aec0784   • android-arm64  • Android 9 (API 28)
    • Chrome     • chrome     • web-javascript • Google Chrome 84.0.4147.105
    • Web Server • web-server • web-javascript • Flutter Tools

• No issues found!
```

**Comment by boatengyeboah** [comment]
Same here

**Comment by letsar** [comment]
Try using the latest Flutter version (1.17.X), or downgrade to 0.3.0

**Comment by Traitor00** [comment]
i am getting same error and i have latest flutter version

**Comment by letsar** [comment]
Hi @Traitor00, this breaking change appeared in stable released 1.17 (see: https://flutter.dev/docs/release/breaking-changes/forgetchild-call-super) that's why I don't think you have the latest stable version. Can you copy/paste the result of `flutter --version` here?
